ethcore/src/json_tests/chain.rs

```diff
@@ -89,16 +90,25 @@ pub fn json_chain_test<H: FnMut(&str, HookType)>(json_data: &[u8], start_stop_ho
 					config.check_seal = false;
 				}
 				config.history = 8;
+				config.queue.verifier_settings.num_verifiers = 1;
```

Comment:
The default is `num_cpus` which includes HT and means we spin up a lot of threads for each test. I think 1 thread is enough here.

Reply:
ok, have you notice any change in execution time? Seems fine

Reply:
Nothing that noticeable I guess. On my machine tests spin up and tear down 12 verifier threads for each test so if nothing else this makes the logs less noisy.